### ISSUE TITLE
The input length and target length of video in denoising_diffusion

### ISSUE THREAD

**Issue by bugsuse** [buggyyang/RVD#4]
I noticed that the loss was calculated when length >= 2. May I ask why is the loss calculated only for values greater than 2? Does this imply that the input consists of 2 frames, and the rest are labels?

https://github.com/buggyyang/RVD/blob/309fbe0d4faeb84f3cfece15d5e85a54f71ba3cb/modules/denoising_diffusion.py#L241-L246

**Comment by wenyufeng936** [comment]
Hello, have you reproduced the results of this paper?

**Comment by buggyyang** [comment]
@bugsuse That's because our video prediction needs context frames as initial condition. These initial condition frames are given so they do not need any loss calculation.

**Comment by buggyyang** [comment]
@wenyufeng936, are you encountering any issues running the test code? You might need to customize the dataloader and adjust the paths to run the model. For other configurations, you can use the provided `config_test.py`.

If you're looking to reproduce the evaluation results, please note there might be some inconsistencies due to the issue discussed in #3. Additionally, the model's performance can be enhanced by training it with more parameters and an improved sampling procedure.

This project represents one of the earliest attempts at video diffusion models. However, due to limited computing resources within our group, its performance might not be as high as the others. This limitation is also why the paper wasn't accepted by a higher-tier conference.
